public/app/plugins/panel/timeseries/migrations.ts

```diff
@@ -314,7 +314,8 @@ export function flotToGraphOptions(angular: any): { fieldConfig: FieldConfigSour
     }
 
     if (angular.legend.values) {
-      options.legend.calcs = getReducersFromLegend(angular.legend);
+      const enabledLegendValues = pickBy(angular.legend);
```

Comment:
super nit: I guess our ambition to minimize the use of `lodash` is not clear to everyone so maybe we should add that to the docs or maybe we should re-evaluate that decision.

Reply:
> super nit: I guess our ambition to minimize the use of `lodash` is not clear to everyone so maybe we should add that to the docs, eslint or maybe we should re-evaluate that decision.

Reply:
Agree with @hugohaggmark , but in case of migrations I think it's fine to add it as lodash is already used in couple of places here. We should definitely avoid importing lodsh in the new code we create.